DP02_13a_Image_Cutout_SciDemo.ipynb

```diff
@@ -0,0 +1,1286 @@
+{
```

Comment:
<p>This circle/polygon discussion is confusing, since a "circle" returns a square image, and a "polygon" returns a rectangle. (I know this is under the hood and not in our control, but the discussion is still confusing...)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>Ok I tried to clarify. Hopefully succeeded in improving it.</p><p> </p>